model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/stencils/compute_edge_diagnostics_for_dycore_and_update_vn.py

```diff
@@ -125,55 +125,37 @@ def _compute_theta_rho_face_values_and_pressure_gradient_and_update_vn(
     fa.EdgeKField[ta.wpfloat],
     fa.EdgeKField[ta.wpfloat],
 ]:
-    (
-        ddx_perturbed_rho,
-        ddy_perturbed_rho,
-        ddx_perturbed_theta_v,
-        ddy_perturbed_theta_v,
-    ) = (
-        _mo_math_gradients_grad_green_gauss_cell_dsl(
-            p_ccpr1=perturbed_rho_at_cells_on_model_levels,
-            p_ccpr2=perturbed_theta_v_at_cells_on_model_levels,
-            geofac_grg_x=geofac_grg_x,
-            geofac_grg_y=geofac_grg_y,
+    # horizontal_pressure_gradient:
+    # rho_at_edges_on_model_levels: ???  (halo exchange afterwards)
+    # next_vn: interior (halo exchange afterwards and lateral boundary afterwards)
+
```

Comment:
main open question: is theta_v read in the halo? (it's not exchanged, unlike rho).
If yes, it might be even better for performance to exchange it together with theta_v and give this program better bounds (assuming latency is a bigger problem in halo exchange than amount of data).